content/letter/terms.md

```diff
@@ -29,15 +29,15 @@ Any information that we provide about the Conference schedule is non-binding and
+We and third parties authorized by us may take photographs, video or audio recordings of the Conference, including of you ( “**Conference Recordings**”). By being permitted to attend the conference (and without additional consideration), you (A) agree that we shall be the sole copyright owner of the Conference Recordings, (B) irrevocably and perpetually consent to the capture of your image, voice and likeness in the Conference Recordings and our right to use, reproduce, distribute, display and publish such Conference Recordings, in whole or in part, for any purpose, at any time and either directly or through any third party, and (C) (on behalf of yourself and your successors and assigns) release us and all such third parties (and us and their respective successors, assigns, and licensees) from any and all liability, claims and causes of action you may have now or in the future with respect to the creation and any exploitation of the Conference Recordings, and (D) agree to pay any attorneys’ fee and costs incurred by us or the third parties if you institute any legal action based on or relating to any consent, waiver, release or other agreement you have given in this paragraph.
 
 ### Your Use of Photography, Recordings and Conference Information
 
 Without our prior written consent, you may not take, publish or disseminate photos, audio recordings and/or video recordings of the Conference or any of its attendees for commercial use, provided that the foregoing does not apply to your personal or internal business use.
 
 ## Tickets
 
-All ticket sales are final and non-refundable. Tickets are non-transferable and may not be assigned without our prior written consent (which may be withheld for any reason or no reason at all) and may only be used by the named individual who purchased the ticket according to our records. Any ticket substitution requests must be submitted to via Universe at least 30 days prior to the start of the Conference.
+All ticket sales are final and non-refundable. Tickets are non-transferable and may only be used by the named individual who purchased the ticket according to our records. Any ticket substitution requests must be submitted to via Universe at least 30 days prior to the start of the Conference.
 
```

Comment:
This also mentions Universe, which I think we want to drop. Also, I don't have a problem with tickets being transferable especially given these time.

"Purchased tickets can be transferred to another individual that accepts our Code of Conduct, Covid Procautions, and Terms and Conditions. Please contact info@satsummit.io to help facilitate transfer or your ticket to a new ticket holder.

Comped tickets that are provided to speakers and others may not be transferred. If you are unable to use your comped ticket, please let us know and we will make sure it is reassigned to another deserving individual."

Reply:
Thanks Ian. I incorporated the language you suggested.